letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -895,9 +895,10 @@ def _add_name_vhost_if_necessary(self, vhost):
         for addr in vhost.addrs:
             # In Apache 2.2, when a NameVirtualHost directive is not
             # set, "*" and "_default_" will conflict when sharing a port
+            addrs = set((addr,))
```

Comment:
Trying to restate what this does for sanity checking:

In addition to ensuring that `addrs` is always defined, this change will cause `add_name_vhost` to be called once for every other vhost with the same address.  Previously it would only be called if there was a `"*"` or `"_default_"` vhost.